policyengine/outputs/macro/single/calculate_single_economy.py

```diff
@@ -49,6 +49,8 @@ class SingleEconomy(BaseModel):
     weekly_hours_substitution_effect: float | None
     type: str
```

Comment:
__issue/suggestion:__ I've amended the type hint, but wonder if cliffs should just be completely removed here and you have two single-economy calculation types: `SingleEconomy` and something that only does cliffs. Curious to hear your thoughts.
```suggestion
    type: Literal["general", "cliff"]
```

Reply:
More detail above, but I think there is a case that adding it in here is cleaner.